lib/mongo_trails/mongo_support/version.rb

```diff
@@ -112,11 +112,13 @@ def object_changes=(value)
     end
 
     def unescape_value(value)
-      value&.deep_transform_keys { |key| URI.unescape(key) }
+      parser = URI::Parser.new
```

Comment:
I think you can create a parser outside of the method not to create it every time